stdlib/cProfile.pyi

```diff
@@ -9,8 +10,10 @@ def runctx(
 
 _SelfT = TypeVar("_SelfT", bound=Profile)
 _T = TypeVar("_T")
+_Label = Tuple[str, int, str]
 
 class Profile:
+    stats: dict[_Label, tuple[int, int, int, int, dict[_Label, tuple[int, int, int, int]]]]  # undocumented
```

Comment:
This should also be added to `profile.pyi`, but `def label` is available only in cProfile.

Reply:
Good catch, added it there too